src/lineage/entrypoint.py

```diff
@@ -352,44 +358,49 @@ def main() -> None:
             logging.warning("Unable to clone %s.", repo.full_name)
             logging.warning(err)
             continue
+            core.end_group()
         lineage_id: str
         local_branch: str
         remote_branch: Optional[str]
         remote_url: str
         if config.get("version") != "1":
-            logging.warning(f"Incompatible config version: {config.get('version')}")
+            logging.warning("Incompatible config version: %s", config.get("version"))
             continue
+            core.end_group()
         if "lineage" not in config:
             logging.warning("Could not find 'lineage' key in config.")
             continue
+            core.end_group()
         for lineage_id, v in config["lineage"].items():
-            logging.info(f"Processing lineage: {lineage_id}")
+            logging.info("Processing lineage: %s", lineage_id)
             try:
                 local_branch = v.get("local-branch")
                 remote_branch = v.get("remote-branch", "HEAD")
                 remote_url = v["remote-url"]
             except KeyError as e:
-                logging.warning(f"Missing config key while reading {lineage_id}: {e}")
+                logging.warning(
```

Comment:
Should we use `logging.error` here?  That would include a stack trace.

Reply:
Do you mean `logging.exception`?